lib/twilio/audiohelper.ts

```diff
@@ -287,6 +292,33 @@ class AudioHelper extends EventEmitter {
     }
   }
 
+  /**
+   * Set whether the disconnect sound is enabled or not
+   * @param isEnabled - Whether the disconnect sound is enabled or not
+   * @returns Whether the disconnect sound is enabled or not
+   */
+  disconnect(isEnabled?: boolean): boolean {
```

Comment:
If we pass `undefined` it seems like these functions just report the enable status of the sound, is that correct? Should we put that behavior in the docstring?

Reply:
Yes that's correct. If you looks at the generated API doc, the return value kind of explains that right? That is, it's always returning the enable status of the sound whether you pass a boolean or undefined.

Reply:
My confusion is that I wasn't sure if passing `undefined` would be inferred as `false` since it's a falsy value, or if it would make it a no-op.

Reply:
It won't be. See `_maybeEnableSound` implementation in this file. I also added tests for this. Please see the tests.

Reply:
Ah did you mean something like `disconnect(undefined)` instead of `disconnect()`?

Reply:
It's clear when reading the implementation but not when reading only the docstring, is what I'm trying to say. Applies to either implicit or explicit `undefined`.

Reply:
Ok gotcha. Can you provide a suggestion on the docstring? What would make it clear?

Reply:
This is too verbose. 
Re: @param doEnable. Typedoc already generates explanation that this is an optional boolean value. I don't think mentioning it again in the docstring is good.
Re: @returns. Typedoc also mentions that the return value is boolean. No need to mention again.
I tweaked it a bit. Please see latest.